### PR TITLE
fix: Added required in TaxId in Billing Info

### LINKED ISSUE #4030: Billing Info form: FE check not working for Company field
**Describe the bug**

- FE check not working for company field
- `Tax ID or business ID` field not  marked as `*`

**Screenshots**
Screenshot from 2020-02-08 18-10-46

**Additional context**

Up For Grabs

### PR DESCRIPTION
**Pull request by Dishebh**
Fixes #4030 

#### Short description of what this resolves:
`Tax Id or Business Id` field now marked as `*`.

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Dishebh** [comment]
> Please See the issue image again, Its actully the billing info part of `order-form` ,
> 
> `open-event-frontend/app/templates/components/forms/orders/order-form.hbs`

Yes, the `Billing info` part of `order form` refers to the `widgets/forms/billing-info`.

`{{#if isBillingInfoNeeded}}   
          {{widgets/forms/billing-info
        	  data=data
          	event=event}} 
 {{/if}}`

**Comment by snitin315** [comment]
I'm not sure, please provide a full-page screenshot of your change.

**Comment by Dishebh** [comment]
> I'm not sure, please provide a full-page screenshot of your change.

Screenshot from 2020-03-15 17-36-10

**Comment by snitin315** [comment]
Add Front-end check for company field as well.

**Comment by Dishebh** [comment]
> Add Front-end check for company field as well.

I'm not sure why the front-end check is not working, since we are already using the `required` class in the `Company` label section as well. 
For now, the pr only solves a part of the issue mentioned above.

**Comment by snitin315** [comment]
Please try to solve the complete issue.

**Comment by Dishebh** [comment]
> Incomplete

This only solves a part of the issue. I am not sure why the check for the `Company` label is not working. That's why, cannot proceed further in solving the complete issue, for now that is.

**Comment by Dishebh** [comment]
What to do about this, then?

**Comment by Dishebh** [comment]
Solved the issue. Pls review

File: app/components/forms/orders/order-form.js [hunk 1]
```diff
@@ -202,7 +202,8 @@ export default Component.extend(FormMixin, {
     };
 
     let companyValidation = {
-      rules: [
+      identifier : 'company',
+      rules : [
```
Comment (niranjan94) [inline app/components/forms/orders/order-form.js]: Codacy Issue found: [Missing space after key 'rules'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5182068)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5182068&bid=17004791)

File: app/components/forms/orders/order-form.js (hunk 1 above)
Comment (Dishebh) [reply app/components/forms/orders/order-form.js]: added

Comment (iamareebjamal) [reply app/components/forms/orders/order-form.js]: Conversations are resolved automatically. Please don't resolve the conversations yourself. The fact that it is not resolved automatically means that the issue is not resolved

**Comment by kushthedude** [comment]
Fix lint and squash commits

**Comment by Dishebh** [comment]
Messed up my branch. Sorry, will close it and open a new one with required changes. Thanks

**Comment by snitin315** [comment]
```
git fetch upstream development
git reset --hard upstream/development
```

**Pull request closed without merging**